controllers/good_morning_controller.py

```diff
@@ -1,60 +1,43 @@
-    last_response: Optional[datetime] = None
+    def get_good_morning_explanation():
+        return GOOD_MORNING_EXPLANATION
 
-    async def _send_accrual_responses(message: Message):
-        response = "\n".join(GoodMorningController.current_accruals)
-        response += MESSAGE_EXPLANATION
-        await message.channel.send(response)
-        GoodMorningController.last_response = message.created_at
-        GoodMorningController.current_accruals = list()
+    async def get_morning_points(interaction: Interaction):
+        points = DB().get_morning_points(interaction.user.id)
+        await interaction.response.send_message(
+            f"Your current weekly count is {points}!", ephemeral=True
+        )
 
-    def _should_respond(message: Message):
-        if GoodMorningController.last_response is None:
-            GoodMorningController.last_response = message.created_at
-
-        if len(GoodMorningController.current_accruals) == 0:
-            return False
-
-        if len(GoodMorningController.current_accruals) >= MESSAGE_BATCH_SIZE:
-            return True
-
-        time_passed = message.created_at - GoodMorningController.last_response
-        if time_passed >= MAX_TIME_BETWEEN_RESPONSE:
-            return True
-
-        return False
-
-    async def handle_response(message: Message):
-        """Respond to good morning messages if requirements met
-
-        Args:
-            message (Message): Message in stream chat
-        """
-        if GoodMorningController._should_respond(message):
-            await GoodMorningController._send_accrual_responses(message)
-
-    async def accrue_good_morning(message: Message):
+    async def accrue_good_morning(interaction: Interaction):
         """Accrue good morning message point
 
         Args:
             message (Message): "good morning" stream chat message
         """
-        accrued = DB().accrue_morning_points(message.author.id)
-        if not accrued:
-            return
+        if interaction.channel.id != STREAM_CHAT_ID:
+            return await interaction.response.send_message(
+                f"You can only say good morning in <#{STREAM_CHAT_ID}>!", ephemeral=True
+            )
 
-        points = DB().get_morning_points(message.author.id)
-        response = f"Good morning {message.author.mention}! Your current weekly count is {points}!"
-        GoodMorningController.current_accruals.append(response)
+        accrued = DB().accrue_morning_points(interaction.user.id)
+        if not accrued:
+            return await interaction.response.send_message(
+                "You've already said good morning today!", ephemeral=True
+            )
+
+        points = DB().get_morning_points(interaction.user.id)
+        await interaction.response.send_message(
+            (
+                f"Good morning {interaction.user.mention}! "
+                f"Your current weekly count is {points}! "
+                f"{GOOD_MORNING_EXPLANATION}"
```

Comment:
this feels gross, can we either:

1. define it as a constant and use `.format()` instead
```py
GOOD_MORNING_MESSAGE_TEMPLATE = f'Good morning {{user}}! Your current weekly count is {{points}}. {GOOD_MORNING_EXPLANATION}'

...

await interaction.response.send_message(
    GOOD_MORNING_MESSAGE_TEMPLATE.format(
        user=interaction.user.mention,
        points=points
    )
)
```
2. join the format strings with `" ".join()`
```py
await interaction.response.send_message(
    " ".join([
        f"Good morning {interaction.user.mention}!",
        f"Your current weekly count is {points}!",
        GOOD_MORNING_EXPLANATION
    ])
)
```

Reply:
I am reasonably confident that the way I have it is the idiomatic approach with format strings. Black only handles this formatting in their `--preview` mode since the AST parsing is new, but this is what it produces given our input/output:

input:
```python
interaction = {}
points = 10 
GOOD_MORNING_EXPLANATION = "TEST"
await interaction.response.send_message(f"Good morning {interaction.user.mention}! Your current weekly count is          {points}! {GOOD_MORNING_EXPLANATION}")
```

output:
```python
interaction = {}
points = 10
GOOD_MORNING_EXPLANATION = "TEST"
await interaction.response.send_message(
    f"Good morning {interaction.user.mention}! Your current weekly count is {points}!"
    f" {GOOD_MORNING_EXPLANATION}"
)
```

I don't like thinking about formatting, so I'd rather stick to Black's preferences here.

Reply:
gross, but correct: https://peps.python.org/pep-0008/#:~:text=The%20preferred%20way%20of%20wrapping%20long%20lines%20is%20by%20using%20Python%E2%80%99s%20implied%20line%20continuation%20inside%20parentheses%2C%20brackets%20and%20braces